docs/userGuide/components/modal.md

```diff
@@ -0,0 +1,15 @@
+# Modal
+
+#### Modals are to be used together with the [Trigger](#trigger) component for activation.
+
+## Modal Options
+Name | type | Default | Description
+--- | --- | --- | ---
+title | `String` | `''` | Title of the Modal component.
+ok-text | `String` | `''` | Text for the OK button.
+effect | `String` | `zoom` | Supports:`zoom` and `fade`.
```

Comment:
Space after the colon.

Also, the rest of the documentation seems to just use comma instead of "and": `Supports: zoom, fade.`